src/Xamarin.Android.Build.Tasks/Linker/MonoDroid.Tuner/FixAbstractMethodsStep.cs

```diff
@@ -19,43 +19,79 @@ namespace MonoDroid.Tuner
 	/// <summary>
 	/// NOTE: this step is subclassed so it can be called directly from Xamarin.Android.Build.Tasks
 	/// </summary>
-	public class FixAbstractMethodsStep : BaseStep
+	public class FixAbstractMethodsStep :
+#if NET5_LINKER
+	BaseMarkHandler
+#else   // !NET5_LINKER
+	BaseStep
+#endif  // !NET5_LINKER
 	{
-		readonly TypeDefinitionCache cache;
```

Comment:
Same comment as with `AddKeepAlivesStep`: this should instead become `IMetadataResolver cache`.  Then the `FixAbstractMethodStep()` constructor wouldn't need *any* changes.

Reply:
Fixed. (I had originally changed the field name to `resolver` in this case, because I didn't want the field on `BaseMarkHandler` to be named `cache`, but it sounds like you prefer to reduce the size of the diff.)